.gitignore

```diff
@@ -18,5 +18,10 @@ packaging/deb/debian/mackerel-agent.bin
 packaging/*.tar.gz
 packaging/deb-build
 packaging/rpm-build
+packaging/*.deb
```

Comment:
Hmm, should we change directory structure to be more gitignore-friendly?

Reply:
It will be nice to put the artifacts in one directory.
But we also need to fix other tools, so let's discuss in another issue.